proposal.md

```diff
@@ -1,31 +1,31 @@
 # Proposal
 
-When a user says:
-My bot will respond by:
+When a user says: tell me a sports score
+My bot will respond by: looking at espn.com and giving that score back
 
-When a user says:
-My bot will respond by:
+When a user says: tell me a joke 
+My bot will respond by: tell the user a random joke
 
-When a user says:
-My bot will respond by:
+When a user says: create an ai image
+My bot will respond by: make an ai image
 
-When a user says:
-My bot will respond by:
+When a user says: set me a __ min reminder for ______
+My bot will respond by: setting that reminder and alarming the user when it is done
 
-When a user says:
-My bot will respond by:
+When a user says: when @user birthday is 
+My bot will respond by: wishing that @user a happy birthday on their birthday
 
-When a user says:
-My bot will respond by:
+When a user says: timer for __ min
+My bot will respond by: setting at timer for that time and going off when the timer has finished
 
-When a user says:
-My bot will respond by:
+When a user says: meme
+My bot will respond by: pasting in a random meme
```

Comment:
Let's stick to text-only tasks, no images. Let's come up with a different idea, or make sure all of these memes are text-based.